ereuse_devicehub/templates/inventory/device_list.html

```diff
@@ -445,5 +454,5 @@ <h5 class="card-title">Documents</h5>
     perPage: 20
   })
 </script>
-<script src="{{ url_for('static', filename='js/main_inventory.build.js') }}"></script>
+<script src="{{ url_for('static', filename='js/main_inventory.js') }}"></script>
```

Comment:
I guess this change is unintentional (revert it!)
```suggestion
<script src="{{ url_for('static', filename='js/main_inventory.build.js') }}"></script>
```